ui/user/src/lib/components/tasks/Task.svelte

```diff
@@ -52,6 +53,17 @@
 	let isTaskInfoVisible = $state(true);
 	let observer: IntersectionObserver;
 
+	let taskRun: Task | undefined = $state(undefined);
+
+	$effect(() => {
+		// Load task run data only in readonly mode; so we can get the correct steps & loop steps related to the current task
```

Comment:
forcing a dumb change to re-trigger build
```suggestion
		// Load task run data only in readonly mode; so that we can get the correct steps & loop steps related to the current task
```